tekton/datadog_checks/tekton/check.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheckV2
+from datadog_checks.tekton.config_models import ConfigMixin
+
+from .metrics import ENDPOINTS_METRICS_MAP
+
+PIPELINES_CONTROLLER_NAMESPACE = 'tekton.pipelines_controller'
+TRIGGERS_CONTROLLER_NAMESPACE = 'tekton.trigger_controller'
```

Comment:
Sorry, I should have asked this before, but where are we using these variables? I must be missing something obvious, but I can't find them anywhere except these declarations.

Reply:
🤦 we don't anymore, I replaced it by [this](https://github.com/DataDog/integrations-core/pull/16665/files/46eda8bc52a15c9e060aa5f849237ed91f433c0e#diff-aacf30ed0d87198de0cee689c664c71f4db3d5bb9e9f1d1cbdec1ae9bc777af4R31)

Good catch

```suggestion
```